config/example/tips.txt

```diff
@@ -168,3 +168,23 @@ You can fax papers between requests consoles by attacking the console with a pap
 No wall or window is 100% impervious to heat.
 There is no anti-security agenda... Or is there?
 Fax machines cannot comprehend the elegant penmanship that is cursive. It is a simple machine afterall.
+As a Xenobiologist, your monkeys are perfectly suitable for human consumption even after your slimes are done with them, perhaps you can work something out with the Chef?
+Never try to fight the hostile maintenance drones on your own.
+Spice up any round by spawning 100 greimorian larva on top of a vent.
+There is no cloning, but you can respawn... as a different person, obviously.
+As a Loner, your psionic powers are a mystery to us all.
+As a Maintenance Drone, you can put on hats, but not take them off.
+As a Maintenance Drone, you cannot repair yourself but you can ask other maintenance drones to do it for you.
+Destroy the bluespace inhibitor for a whacky, wild adventure.
+The Maintenance Drones will never threaten to stab you, and in fact, cannot speak.
+The Bluespace Technicians aren't real, they can't hurt you.
+There is no escape.
+As a Dionaea, you can store items in your stomach, including guns.
```

Comment:
```suggestion
As a Diona, you can store items in your stomach, including guns.
```

Reply:
I based this on line 146 and 132 do you want me to update those as well?

Reply:
yuh. singular for diona is diona, plural is dionae, change as ya see fit

Reply:
ahhh I understand